app/boards/arm/glove80/glove80_rh_defconfig

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2021 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_GLOVE80_RH=y
+
+# Keyboard IDs
+CONFIG_ZMK_KEYBOARD_NAME="Glove80 Right"
```

Comment:
Doesn't this accidentally cause the right hand to advertise in a way that's visible to hosts? Seems like this would invite accidental disconnects. If you just need this for USB, set just the USB Kconfig you need?

Reply:
Empirically, it doesn't show up for either MacOS or Windows. But also that's definitely something we'd want to avoid. Do you recommend removing it and using only `CONFIG_USB_DEVICE_PRODUCT` for future-compatibility purposes?

The intent was for the name to be defined when scanning for Bluetooth devices using lower level tools like nRF Connect.

Reply:
Out of band:
> As long as the lack of other GAP attributes or advertised services is enough for hosts not to show it still, fine to leave it.